compiler/crates/relay-compiler/src/compiler_state.rs

```diff
@@ -341,7 +341,7 @@ impl CompilerState {
                 .any(|sources| !sources.processed.is_empty())
     }
 
-    fn is_change_safe(&self, sources: &SchemaSources) -> bool {
+    fn is_change_safe(&self, sources: &SchemaSources, project_config: &ProjectConfig) -> bool {
```

Comment:
can we add `node_interface_id_field` to the compiler_state itself?

Reply:
I tried to do this, but the config that's used to create the compiler_state is the root config, not the project config, so this seemed challenging to try to move.